### PR TITLE
Add flag for setting logging verbosity.

### PR DESCRIPTION
**Pull request by elliotwaite**
Adds --log flag to callable scripts.
Valid values: DEBUG, INFO (default), WARN, ERROR, or FATAL.

Example usage:

bazel build //magenta/models/basic_rnn:basic_rnn_create_dataset

./bazel-bin/magenta/models/basic_rnn/basic_rnn_create_dataset \
--input=/tmp/notesequences.tfrecord \
--output_dir=/tmp/basic_rnn/sequence_examples \
--eval_ratio=0.10 \
--log=DEBUG

### CODE REVIEW

File: magenta/models/shared/melody_rnn_generate.py [hunk 1]
```diff
@@ -147,7 +149,7 @@ def run(melody_encoder_decoder, build_graph):
         (mode, hparams_string, input_size, num_classes, sequence_example_file).
         For an example usage, see models/basic_rnn/basic_rnn_graph.py.
   """
-  tf.logging.set_verbosity(tf.logging.INFO)
+  tf.logging.set_verbosity(FLAGS.log)
```
Comment (danabo) [inline magenta/models/shared/melody_rnn_generate.py]: set_verbosity takes a string?

**Comment by danabo** [comment]
LGTM.
We will have to remember to do this for every binary. Pretty annoying.

File: magenta/models/shared/melody_rnn_generate.py (hunk 1 above)
Comment (elliotwaite) [inline magenta/models/shared/melody_rnn_generate.py]: Yeah, it can take an int, long, or string.

**Comment by cghawthorne** [comment]
LGTM

**Comment by cghawthorne** [comment]
This should fix #115, right?

**Comment by danabo** [comment]
Not quite. The actual logging statements were removed.

**Comment by danabo** [comment]
#115 is fixed. To update this PR, add a `log` flag to this file where tf.logging.set_verbosity is called: https://github.com/tensorflow/magenta/blob/master/magenta/models/shared/melody_rnn_create_dataset.py#L95

**Pull request merged**